slick.grid.js

```diff
@@ -3441,7 +3441,7 @@ if (typeof Slick === "undefined") {
 
     function updateRowPositions() {
       for (var row in rowsCache) {
-        rowsCache[row].rowNode.style.top = getRowTop(row) + "px";
+        rowsCache[parseRowToNumber(row)].rowNode[0].style.top = getRowTop(parseRowToNumber(row)) + "px";
```

Comment:
Changing `rowNode` to `rowNode[0]` is the biggest change I see here and I'm not sure if this is always valid. Have you tried with a few of the examples to see if any errors comes up?

Reply:
At this point I'm confused by whether the ```rowNode``` is a jQuery object. If it is, then ```rowNode[0]``` will return the DOM node.

Also, I'd be happy to ditch the function and just put 

    var rn = row ? parseInt(row) : 0;

inside the loop. That stops calculating twice.

Reply:
Sorry for not calling this out sooner but yes the biggest change that I saw was that rowNode was a jQuery object when it was erroring out. Hence the top of undefined error. 

But also `row` from rowsCache returns a string but the rowsCache key is a number. Parsing as an integer ensures we were grabbing the correct value from `rowsCache`.

I will make the changes per @6pac's suggestion and revert the changes for the function that was created. (Was only a small refactoring but not needed).

Thanks for looking at this gentlemen.

Reply:
Sorry for not calling this out sooner but yes the biggest change that I saw was that rowNode was a jQuery object when it was erroring out. Hence the top of undefined error.

But also row from rowsCache returns a string but the rowsCache key is a number. Parsing as an integer ensures we were grabbing the correct value from rowsCache.

I will make the changes per @6pac's suggestion and revert the changes for the function that was created. (Was only a small refactoring but not needed).

Thanks for looking at this gentlemen.